hedera-node/src/test/java/com/hedera/services/txns/crypto/CryptoUpdateTransitionLogicTest.java

```diff
@@ -95,28 +100,49 @@
 	private static final String MEMO = "Not since life began";
 
 	private boolean useLegacyFields;
+//	private HederaLedger ledger;
+//	private OptionValidator validator;
+//	private TransactionBody cryptoUpdateTxn;
+//	private TransactionContext txnCtx;
+//	private PlatformTxnAccessor accessor;
+//	private CryptoUpdateTransitionLogic subject;
+//	private GlobalDynamicProperties dynamicProperties;
+
+	@Mock
 	private HederaLedger ledger;
-	private OptionValidator validator;
+	@Mock
+	private AccountStore accountStore;
+	@Mock
+	private EntityIdSource ids;
+	@Mock
+	private ContextOptionValidator validator;
+
 	private TransactionBody cryptoUpdateTxn;
+	@Mock
 	private TransactionContext txnCtx;
+	@Mock
 	private PlatformTxnAccessor accessor;
+
 	private CryptoUpdateTransitionLogic subject;
+	@Mock
 	private GlobalDynamicProperties dynamicProperties;
+	@Mock
+	private TransactionRecordService transactionRecordService;
 
 	@BeforeEach
 	private void setup() {
```

Comment:
Since there are a lot of changes to the transition logic, I was expecting more changes in this file and for sure some old unit tests that no longer work/are applicable should be removed.

Reply:
unit test are WIP